### DISCUSSION TITLE
Support for zsh-z

### DISCUSSION THREAD

**Discussion by hacker-DOM** [sxyazi/yazi#757] [4. Ideas]
It would be cool if yazi supported [zsh-z](https://github.com/agkozak/zsh-z) alongside zoxide.

**Comment by sxyazi** [comment]
Hi, I've now reimplemented zoxide as a plugin in https://github.com/sxyazi/yazi/issues/865, which will allow users to copy it as a template and easily modify the commands inside.

That is, you can copy https://github.com/sxyazi/yazi/blob/main/yazi-plugin/preset/plugins/zoxide.lua as your `~/.config/yazi/plugins/zoxide.yazi/init.lua`, and then change the `zoxide` command inside with `zsh-z`, and Yazi will load it automatically
